js/GraphicView.js

```diff
@@ -2,19 +2,74 @@ import ComponentView from 'core/js/views/componentView';
 
 class GraphicView extends ComponentView {
 
+  className() {
+    return [
+      super.className(),
+      this.model.get('_isScrollable') && 'is-scrollable'
+    ].filter(Boolean).join(' ');
+  }
+
   events() {
     return {
-      'click .js-graphic-link': 'onClick'
+      'click .js-graphic-link': 'onClick',
+      'keydown .js-graphic-scrollbar': 'onKeyDown'
     };
   }
 
   postRender() {
     this.$('.graphic__widget').imageready(() => {
       this.setReadyStatus();
       this.setupInviewCompletion('.graphic__widget');
+      this.setupScrollable();
     });
   }
 
+  setupScrollable() {
+    if (!this.model.get('_isScrollable')) return;
+    this.onScroll = _.debounce(this.onScroll.bind(this), 17);
```

Comment:
Why 17?

Reply:
1000/60fps = 16.666

Reply:
@oliverfoster when testing and scrolling quickly the graphic seems to jolt occasionally, I lowered this to 14 and it seemed to sort the issue on my screen at least, it may still be an issue that appears on some screens.

Reply:
Yea, it will jolt. Too much javascript going off at one time. It's not just this code listening to scroll events in Adapt. Best not to expect and update timeout higher than the frame rate, it'll just block up the callback queue more.